content/blog/tidymodels-2024-survey/index.Rmd

```diff
@@ -0,0 +1,63 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2024-survey
+title: Take the tidymodels survey for 2024 priorities
+date: 2024-02-26
+author: Emil Hvitfeldt
+description: >
+    We are conducting our third tidymodels priorities survey. Please give us your
+    feedback!
+
+photo:
+  url: https://unsplash.com/photos/white-flowers-under-blue-sky-during-daytime-peN6l68AWaw
+  author: Aamyr
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [other] 
+tags: [survey,tidymodels]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+At the end of 2021, we created a survey to get community input on how we prioritize our projects. [The results](https://colorado.posit.co/rsc/tidymodels-priorities-2022/) gave us a good sense of which items people were most interested in. Since then we have completed a number of projects:
+
+* **Model fairness metrics** were included in [yardstick 1.3.0](https://yardstick.tidymodels.org/news/index.html#yardstick-130) with [tidymodels.org](https://www.tidymodels.org/) posts coming soon.
+* **Spatial analysis models and methods** led to the creation of [spatialsample](https://spatialsample.tidymodels.org/).
+* **H2O.ai support** was achieved with the creation of [agua](https://agua.tidymodels.org/).
+* **Better serialization tools** Serialization of tidymodels objects can now be done with [bundle](https://github.com/rstudio/bundle).
+
+Almost everything that respondents prioritized highly last year has either been completed or is currently in progress. Our main focus right now is to wrap up survival analysis, which is being done right now with a series of CRAN releases for the affected packages. The upcoming things we will be working on are postprocessing and supervised feature selection. Since this survey seemed to work well last year, we've decided to use it again to prioritize features for next year.
+
+## Looking toward 2024
+
+**Take a look at [our survey for next priorities](TODO ADD LINK HERE)** and let us know what you think. There are some items we've put "on the menu" but you can write in other items that you are interested in. 
+
+The current slate of our possible priorities include: 
```

Comment:
Let's make these subsections that we can link to. I want the actual survey to have links to these sections.

Reply:
You want each bullet to be a section?

as in `sparse tibbles` be a section, `Causal inference interface` be a section, etc etc.

Reply:
Also, from looking at the survey preview, I don't think we can link to each section from each bullet in the survey

Reply:
> You want each bullet to be a section?

Subsection, yes. 

> Also, from looking at the survey preview, I don't think we can link to each section from each bullet in the survey

We can't use hyperlinks there in bullets?

Reply:
Done!